### PR TITLE
Accessibility additions

### PR DESCRIPTION
**Pull request by Darkshadenl**
**Closes #XX**

## Description

Just a few lines of accessibility hints in different places.

## Checklist

- [x] I have run `yarn format`
- [x] I have run `yarn eslint` and fixed any issues
- [x] This PR does not add any errors to the console

A build will automatically be run by GitHub actions when any changes are made on this PR. This must complete successfully before merging.

## Screenshots or videos

<details>
<summary>Click to expand</summary>

</details>

### CODE REVIEW

**Review by GewoonJaap — APPROVED** [review]

File: src/components/questions/AudioRecorder.tsx [hunk 1]
```diff
@@ -131,28 +138,68 @@ const AudioRecorder = (props: { onAudioRecorded: (recordUri: string) => void })
       <View style={styles.container}>
         <View style={styles.rowContainer}>
           {isRecording ? (
-            <Icon name="stop" size={48} style={styles.icon} onPress={onStopRecord} />
+            <Icon
+              name="stop"
+              size={48}
+              accessibilityLabel={'Stop opname knop'}
```
Comment (MartijnAmbagtsheer) [inline src/components/questions/AudioRecorder.tsx]: "Spraakopname stoppen knop".

File: src/components/questions/AudioRecorder.tsx [hunk 2]
```diff
@@ -131,28 +138,68 @@ const AudioRecorder = (props: { onAudioRecorded: (recordUri: string) => void })
...
+              style={styles.icon}
+              onPress={onStopRecord}
+            />
           ) : (
-            <Icon name="microphone" size={48} style={styles.icon} onPress={onStartRecord} />
+            <Icon
+              name="microphone"
+              accessibilityLabel={'Opnameknop'}
+              accessibilityHint={'Spreek uw antwoord in na het activeren van de knop'}
```
Comment (MartijnAmbagtsheer) [inline src/components/questions/AudioRecorder.tsx]: "Start spraakopname knop"

File: src/components/questions/AudioRecorder.tsx [hunk 3]
```diff
@@ -131,28 +138,68 @@ const AudioRecorder = (props: { onAudioRecorded: (recordUri: string) => void })
...
+              style={styles.icon}
+              onPress={onStartRecord}
+            />
           )}
           {isPlaying ? (
-            <TouchableOpacity disabled={isDisabled} activeOpacity={0.5} onPress={onPausePlay}>
+            <TouchableOpacity
+              disabled={isDisabled}
+              accessible={true}
+              accessibilityLabel={'Pauzeer knop'}
```
Comment (MartijnAmbagtsheer) [inline src/components/questions/AudioRecorder.tsx]: "pauzeer spraakopname knop"

File: src/components/questions/AudioRecorder.tsx [hunk 4]
```diff
@@ -131,28 +138,68 @@ const AudioRecorder = (props: { onAudioRecorded: (recordUri: string) => void })
...
+            >
               <Icon name="pause" size={48} style={isDisabled ? styles.disabled : styles.icon} />
             </TouchableOpacity>
           ) : (
-            <TouchableOpacity disabled={isDisabled} activeOpacity={0.5} onPress={onStartPlay}>
+            <TouchableOpacity
+              disabled={isDisabled}
+              activeOpacity={0.5}
+              accessible={true}
+              accessibilityLabel={'Afspeel knop'}
```
Comment (MartijnAmbagtsheer) [inline src/components/questions/AudioRecorder.tsx]: "Spraakopname afspelen knop"

File: src/components/questions/AudioRecorder.tsx [hunk 5]
```diff
@@ -131,28 +138,68 @@ const AudioRecorder = (props: { onAudioRecorded: (recordUri: string) => void })
...
+          >
             <Icon name="backward" size={48} style={isDisabled ? styles.disabled : styles.icon} />
           </TouchableOpacity>
 
-          <TouchableOpacity disabled={isDisabled} activeOpacity={0.5} onPress={onRemoveRecord}>
+          <TouchableOpacity
+            disabled={isDisabled}
+            activeOpacity={0.5}
+            accessibilityLabel={'Verwijderknop'}
+            accessibilityHint={'Verwijderd opname'}
```
Comment (MartijnAmbagtsheer) [inline src/components/questions/AudioRecorder.tsx:193]: "Verwijder spraakopname knop"

File: src/components/questions/AudioRecorder.tsx [hunk 6]
```diff
@@ -131,28 +138,68 @@ const AudioRecorder = (props: { onAudioRecorded: (recordUri: string) => void })
...
               <Icon name="play" size={48} style={isDisabled ? styles.disabled : styles.icon} />
             </TouchableOpacity>
           )}
-          <TouchableOpacity disabled={isDisabled} activeOpacity={0.5} onPress={onStopPlay}>
+          <TouchableOpacity
+            disabled={isDisabled}
+            activeOpacity={0.5}
+            onPress={onStopPlay}
+            accessibilityLabel={'Terugspoelknop'}
+            accessibilityHint={'Ga terug naar begin opname'}
```
Comment (MartijnAmbagtsheer) [inline src/components/questions/AudioRecorder.tsx:184]: "Spraakopname starten vanaf het begin knop"

File: src/screens/SectionScreen.tsx [hunk 7]
```diff
@@ -47,7 +47,13 @@ const SectionScreen = () => {
         <>
           <View>
             <Text style={styles.questionsTitle}>Vragen</Text>
-            <Text style={styles.questionsAnswered}>
+            <Text
+              style={styles.questionsAnswered}
+              accessible={true}
+              accessibilityLabel={`${determineProgress(section.questions)} van de ${
+                section.questions.length
+              } beantwoord.`}
```
Comment (MartijnAmbagtsheer) [inline src/screens/SectionScreen.tsx]: "van de ... vragen beantwoord"

**Review by MartijnAmbagtsheer — CHANGES_REQUESTED** [review]
Iets meer consistentie inbouwen.

**Review by MartijnAmbagtsheer — APPROVED** [review]

**Pull request merged**